axis.go

```diff
@@ -377,17 +378,22 @@ func (DefaultTicks) Ticks(min, max float64) (ticks []Tick) {
 	}
 	majorDelta := float64(majorMult) * tens
 	val := math.Floor(min/majorDelta) * majorDelta
-	prec := precisionOf(majorDelta)
 	for val <= max {
 		if val >= min && val <= max {
```

Comment:
The second expression here is always true.

Reply:
Removed second `val <= max`, the rest of the condition is valuable.